docs/src/docs-table.tsx

```diff
@@ -0,0 +1,238 @@
+import {
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  mediaQueries as mq,
+} from "@kiwicom/orbit-components";
+import { StyledTable } from "@kiwicom/orbit-components/lib/Table";
+import { StyledTableCell } from "@kiwicom/orbit-components/lib/Table/TableCell";
+import { StyledTableRow } from "@kiwicom/orbit-components/lib/Table/TableRow";
+import { StyledTableBody } from "@kiwicom/orbit-components/lib/Table/TableBody";
+
+export const TableHeadCell = styled(StyledTableCell)`
+  background-color: ${({ theme }) => theme.orbit.paletteCloudLight};
+  text-align: left;
+`;
+
+const Table = styled(StyledTable)`
+  border-collapse: separate;
+  border-spacing: 0;
+
+  ${StyledTableCell} {
+    border-width: 1px;
+    border-color: ${({ theme }) => theme.orbit.borderColorTable};
+    border-style: none none solid none;
+  }
+
+  ${TableHeadCell} {
+    border-style: solid none none none;
+
+    &:first-child {
+      border-top-left-radius: 6px;
+      border-left-style: solid;
+    }
+
+    &:last-child {
+      border-top-right-radius: 6px;
+      border-right-style: solid;
+    }
+  }
+  ${StyledTableBody} > ${StyledTableRow} {
+    transition: transform ease-in-out 0.2s;
+
+    &:hover {
+      background-color: inherit;
+
+      ${mq.desktop(css`
+        border-radius: 6px;
+        transform: scale(1.025);
+        box-shadow: 0px 4px 8px 0px #252a311f, 0px 1px 4px 0px #252a3129;
+
+        ${StyledTableCell} {
+          border-top-style: solid;
+
+          &:first-child {
+            border-radius: 6px 0 0 6px;
+          }
+
+          &:last-child {
+            border-radius: 0 6px 6px 0;
+          }
+        }
+      `)}
+    }
+
+    ${StyledTableCell} {
+      &:first-child {
+        border-left-style: solid;
+      }
+
+      &:last-child {
+        border-right-style: solid;
+      }
+    }
+
+    &:last-child {
+      ${StyledTableCell} {
+        &:first-child {
+          border-bottom-left-radius: 6px;
+        }
+
+        &:last-child {
+          border-bottom-right-radius: 6px;
+        }
+      }
+    }
+  }
+`;
+
+const PropsTableHead = ({ tableHeaders }) => {
+  return (
+    <TableHead>
+      <TableRow>
+        {tableHeaders.map(th => (
+          <TableHeadCell>{th}</TableHeadCell>
+        ))}
+      </TableRow>
+    </TableHead>
+  );
+};
+
+const PropsTableBody = ({ tableHeaders, tableData }) => {
+  return (
+    <TableBody>
+      {tableData.map(row => {
+        return (
+          <TableRow>
+            {tableHeaders.map(header => {
+              return <TableCell>{row[header]}</TableCell>;
+            })}
+          </TableRow>
+        );
+      })}
+    </TableBody>
+  );
+};
+
+const StyledTableOuter = styled.div<{
+  showShadows: boolean;
+  showRight: boolean;
+  showLeft: boolean;
+}>`
+  max-width: 100%;
+  width: 100%;
+  position: relative;
+
+  &::after,
+  &::before {
+    content: " ";
+    display: ${({ showShadows }) => (showShadows ? "block" : "none")};
```

Comment:
just a small detail, as we discussed before, to look it cleaner - use single destructure on the top ;)